examples/fast_style_transfer/sketch.js

```diff
@@ -0,0 +1,49 @@
+/*
+===
+Fast Style Transfer Simple Example
+===
+*/
+
+let net;
+let inputImg;
+let outputImgData;
+
+function setup() {
+  createCanvas(252, 252);
+  net = new p5ml.TransformNet(modelLoaded, 'udnie', 'models/udnie/');
```

Comment:
Is there an option to use this with `preload()`? Also I think the convention is to have the callback last?  Is it possible to load more than one style model into the same `p5ml.TransformNet` instance? Is that way a name is required? Would it be simpler to just create multiple `p5ml.TransformNet` instances with different model directories?

Reply:
It seems like [p5's `preload()`](https://github.com/processing/p5.js/blob/1acf1e332c57c74f9b899cb28742a7a8809d7406/src/core/core.js#L574) only waits for its native methods, like `loadImage()`. I think using a `callback` might be more generic, and works outside of p5.js.

It's possible to load more than one style model in one instance, but it's simpler to create multiple `p5ml.TransformNet` instance, and a name is not required in this case. so it will be `net1 = new p5ml.TransformNet('models/udnie/', callback)` and `net2 = new p5ml.TransformNet('models/wave/', callback)`.